.github/workflows/chromatic.yml

```diff
@@ -0,0 +1,40 @@
+# .github/workflows/chromatic.yml
+
+# Workflow name
+name: 'Chromatic'
+
+# Event for the workflow
+on: 
+  pull_request:
+    branches: [ "main" ]
+    types: [opened, synchronize, reopened, closed]
+  push:
+    branches: [ "main" ]
+
+env:
+  FIGMA_TOKEN: ${{ secrets.FIGMA_AUTH_TOKEN }}
+  FONT_AWESOME_AUTH_TOKEN: ${{ secrets.FONT_AWESOME_AUTH_TOKEN }}
+
+
+# List of jobs
+jobs:
+  chromatic-deployment:
+    # Operating System
+    runs-on: ubuntu-latest
+    # Only run when the PR is out of the draft phase
+    if: '! github.event.pull_request.draft'
+    # Job steps
+    steps:
+      - uses: actions/checkout@v1
+      - name: Install dependencies
+        # 👇 Install dependencies with the same package manager used in the project (replace it as needed), e.g. yarn, npm, pnpm
+        run: yarn
+        # 👇 Adds Chromatic as a step in the workflow
+      - name: Publish to Chromatic
+        uses: chromaui/action@v1
+        # Chromatic GitHub Action options
+        with:
+          # 👇 Chromatic projectToken, refer to the manage page to obtain it.
+          projectToken: ${{ secrets.CHROMATIC_TOKEN }}
```

Comment:
Why not have the `CHROMATIC_TOKEN` in the `env` as well? (similar to the other workflows)

Reply:
not sure that's going to work; we need to pass the project ID along with the script so it can link it to the correct project (we have multiple now)